src/status_im/ui/screens/wallet/add_new/views.cljs

```diff
@@ -126,39 +127,48 @@
           (re-frame/dispatch [:set-in [:add-account :account-error] nil])
           (re-frame/dispatch [:set-in [:add-account :private-key] (security/mask-data %)]))}])])
 
+(defview pin []
+  (letsubs [pin         [:hardwallet/pin]
+            status      [:hardwallet/pin-status]
+            error-label [:hardwallet/pin-error-label]]
+    [react/keyboard-avoiding-view {:style {:flex 1}}
+     [topbar/topbar]
+     [pin.views/pin-view
+      {:pin               pin
+       :status            status
+       :title-label       :t/current-pin
+       :description-label :t/current-pin-description
+       :error-label       error-label
+       :step              :export-key}]]))
+
 (defview add-account []
   (letsubs [{:keys [type account] :as add-account} [:add-account]
             add-account-disabled? [:add-account-disabled?]
-            entered-password      (reagent/atom "")]
+            entered-password      (reagent/atom "")
+            keycard?              [:keycard-multiaccount?]]
     [react/keyboard-avoiding-view {:style {:flex 1}}
      [add-account-topbar type]
      [react/scroll-view {:keyboard-should-persist-taps :handled
                          :style                        {:flex 1}}
-      [settings add-account entered-password]
+      (when-not keycard?
+        [settings add-account entered-password])
       [common-settings account]]
      [toolbar/toolbar
       {:show-border? true
        :right
        {:type                :next
         :label               :t/add-account
         :accessibility-label :add-account-add-account-button
-        :on-press            #(re-frame/dispatch [:wallet.accounts/add-new-account
-                                                  (ethereum/sha3 @entered-password)])
-        :disabled?           (or add-account-disabled?
-                                 (and
-                                  (not (= type :watch))
-                                  (not (spec/valid? ::multiaccounts.db/password @entered-password))))}}]]))
+        :on-press
+        #(re-frame/dispatch [:hardwallet/new-account-pin-sheet
```

Comment:
what about regular account?

Reply:
Yep right, I just thought of merging this pr and creating an issue like #10101 for a regular multiacc. Will fix soon

Reply:
fixed